src/Stratis.Bitcoin.Features.Wallet/WalletManager.cs

```diff
@@ -345,6 +346,42 @@ public Wallet RecoverWallet(string password, string name, string mnemonic, DateT
                 this.UpdateWhenChainDownloaded(new[] { wallet }, creationTime);
             }
 
+            this.SaveWallet(wallet);
+            this.Load(wallet);
+
+            this.logger.LogTrace("(-)");
+            return wallet;
+        }
+
+        /// <inheritdoc />
+        public Wallet RecoverWallet(string name, ExtPubKey extPubKey, int accountIndex, DateTime creationTime)
+        {
+            Guard.NotEmpty(name, nameof(name));
+            Guard.NotNull(extPubKey, nameof(extPubKey));
+            this.logger.LogTrace("({0}:'{1}')", nameof(name), name);
```

Comment:
Entry log should log all input parameters

Reply:
ok,